form/unit_testing.rst

```diff
@@ -156,12 +156,31 @@ make sure the ``FormRegistry`` uses the created instance::
         }
     }
 
+Forms Using Validation
+----------------------
+
+If your forms uses the ``invalid_message`` or ``constraints`` option for validation, you need to
+register the validation extension which provides this options.
+Luckily Symfony provides a custom test class which does this for you.
```

Comment:
I would suggest `Luckily Symfony provides a base test class which takes care of it, just extend [class].`

And get rid of the following sentence, what do you think?